<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTab</class>
 <widget class="QWidget" name="DataTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutDataConfig">
     <item>
      <widget class="QLabel" name="labelDataTimeFrom">
       <property name="text">
        <string>From</string>
       </property>
       <property name="buddy">
        <cstring>editDataTimeFrom</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="editDataTimeFrom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2016</year>
         <month>11</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerData1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDataTimeTo">
       <property name="text">
        <string>To</string>
       </property>
       <property name="buddy">
        <cstring>editDataTimeTo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="editDataTimeTo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2016</year>
         <month>11</month>
         <day>1</day>
        </datetime>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerData2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDataStep">
       <property name="text">
        <string>Step</string>
       </property>
       <property name="buddy">
        <cstring>spinDataStep</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinDataStep">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonShow">
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="widgetData">
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab-bar { alignment: center; } </string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDemand">
      <attribute name="title">
       <string>Demand Pattern</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Select all / Clear</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Set</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QQuickWidget" name="containerGrid">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>300</height>
            </size>
           </property>
           <property name="resizeMode">
            <enum>QQuickWidget::SizeRootObjectToView</enum>
           </property>
           <property name="source">
            <url>
             <string>qrc:/Maps/map.qml</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <layout class="QGridLayout" name="containerDemandTime"/>
       </item>
       <item row="0" column="2">
        <widget class="Line" name="lineDemand">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDistribution">
      <attribute name="title">
       <string>Distribution</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="containerDistributionTime"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRevenue">
      <attribute name="title">
       <string>Revenue</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QGridLayout" name="containerRevenue"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonShow</sender>
   <signal>clicked()</signal>
   <receiver>DataTab</receiver>
   <slot>updatePlots()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updatePlots()</slot>
 </slots>
</ui>
